lib/active_merchant/billing/gateways/orbital.rb

```diff
@@ -219,6 +219,7 @@ def purchase(money, creditcard, options = {})
             add_customer_data(xml, creditcard, options)
             add_managed_billing(xml, options)
           end
+          add_card_indicators(xml, options) if @options[:card_indicators]
```

Comment:
Glancing over the Jira ticket, it seems like this field might also need to be added to the `authorize` action above.

Reply:
Done! Thank you for catching that.